### DISCUSSION TITLE
sequence_tolerance

### DISCUSSION THREAD

**Discussion by musen-zhang** [RosettaCommons/rosetta#226] [ROSIE]
![1](https://github.com/user-attachments/assets/82980b57-3b16-4bec-88d2-d9651e5b71f1)
![2](https://github.com/user-attachments/assets/3c62dbd2-23d6-42ed-ad66-499089107a73)
Why can't I generate a sequence diagram for my amino acid mutation?

**Comment by lyskov** [comment]
Out of scope and non-actionable at this time — closing for now.
